### PR TITLE
added neovim v0.2.2

### PR DESCRIPTION
**Pull request by prabirshrestha**
//cc @justinmk

```cmd
scoop install neovim
```

### CODE REVIEW

File: bucket/neovim.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+            "hash": "66bab53ec0a8aff854921b6925630026649f3a49bb7913d19a4a602ffc138c92"
+        }
+    },
+    "extract_dir": "Neovim",
+    "bin": [
+        "bin\\nvim.exe",
+        "bin\\nvim-qt.exe"
+    ],
+    "checkver": {
+        "github": "https://github.com/neovim/neovim"
```
Comment (justinmk) [inline bucket/neovim.json]: https://github.com/neovim/neovim/releases/latest redirects to the latest tag, maybe that's useful.

**Comment by r15ch13** [comment]
Neovim already exists in the extras bucket: [neovim.json](https://github.com/lukesampson/scoop-extras/blob/master/neovim.json) / [neovim-qt.json](https://github.com/lukesampson/scoop-extras/blob/master/neovim-qt.json)

File: bucket/neovim.json (hunk 1 above)
Comment (r15ch13) [reply bucket/neovim.json]: The `github` utilizes the `/releases/latest` part 👍

**Comment by prabirshrestha** [comment]
@r15ch13 Given that neovim now officially supports windows can we bring it to the main scoop bucket instead of extras?

**Comment by r15ch13** [comment]
@prabirshrestha done 😄

**Comment by prabirshrestha** [comment]
Thanks!

**Comment by r15ch13** [comment]
FYI: `neovim-qt` is now removed from the extras bucket: https://github.com/lukesampson/scoop-extras/commit/0d245769dce5f126a74aeca7e37816515ced2aa4

**Pull request closed without merging**